src/BlockCore.Networks.Cybits/Networks/Policies/CybitsStandardScriptsRegistry.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+using Blockcore.Consensus.ScriptInfo;
+using Blockcore.Consensus.TransactionInfo;
+using Blockcore.Networks;
+using NBitcoin.BitcoinCore;
+
+namespace Blockcore.Networks.Cybits.Policies
+{
+   /// <summary>
+   /// Blockcore sample coin-specific standard transaction definitions.
+   /// </summary>
+   public class CybitsStandardScriptsRegistry : StandardScriptsRegistry
+   {
+      // See MAX_OP_RETURN_RELAY in stratisX, <script.h>
+      public const int MaxOpReturnRelay = 40;
```

Comment:
this should probably be bigger maybe like in bitcoin 82 bytes

Reply:
updated to 83